### PR TITLE
Update submodule pointer for Kernel V10.6.0

### PR DESCRIPTION
**Pull request by kar-rahul-aws**
Description
-----------
Update submodule pointer for FreeRTOS Kernel V10.6.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aggarg — APPROVED** [review]

**Review by shubnil — DISMISSED** [review]

File: manifest.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: "This is the standard distribution of FreeRTOS."
 
 dependencies:
   - name: "FreeRTOS-Kernel"
-    version: "d3c289f"
+    version: "54b135"
```
Comment (sirnish) [inline manifest.yml:7]: Which commit ID this is pointing to?

**Review by sirnish — DISMISSED** [review]
GTG

**Review by moninom1 — DISMISSED** [review]

File: manifest.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: "This is the standard distribution of FreeRTOS."
 
 dependencies:
   - name: "FreeRTOS-Kernel"
-    version: "d3c289f"
```
Comment (moninom1) [inline manifest.yml:7]: The top commit in the submodule updated is c3dc20f, there seems to be mismatch, is that correct?

File: manifest.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ description: "This is the standard distribution of FreeRTOS."
...
+    version: "54b1356"
```
Comment (moninom1) [inline manifest.yml:7]: The top commit in the submodule updated is c3dc20f, there seems to be mismatch, is that correct?

Comment (kar-rahul-aws) [reply manifest.yml]: Updated .

**Review by chinglee-iot — APPROVED** [review]

**Review by ActoryOu — APPROVED** [review]

**Pull request merged**